kebechet/kebechet_runners.py

```diff
@@ -172,11 +172,26 @@ def run(
         return
 
     managers = config.managers
-
+    if config.config.get("overlays_dir"):
+        runtime_environment = [
+            e["name"] for e in config.config.get("runtime_environments")
+        ]
+    else:
+        runtime_environment = [config.config["runtime_environments"][0]["name"]]
     for manager in managers:
         # We do pops on dict, which changes it. Let's create a soft duplicate so if a user uses
         # YAML references, we do not break.
         manager = dict(manager)
+        disallow_run_times = manager.get("disallow", None)
+        allowed_run_times = manager.get("allow", None)
```

Comment:
These are technically mutually exclusive. Not sure how important it is to users though.